### PR TITLE
[DO NOT MERGE UNTIL TESTED] IT'S UPSTREAM MERGE TIME

### PR DESCRIPTION
**Pull request by ghost**
Your welcome for MANUALLY GOING THROUGH 250 CONFLICTS

Only change i've made is a modular revert to https://github.com/HippieStation/HippieStation/pull/11754, due to it's status of -1

Here's your list of errors: https://paste.ee/p/OnNa9

### CODE REVIEW

**Comment by yoyobatty** [comment]
reagent id's need to be ported. Lmao.

**Comment by ghost** [comment]
LUCKILY I GOT A SCRIPT THAT FIXES RECIPES

**Comment by ghost** [comment]
#### Test Merge Deployed

##### Server Instance
HippieStation

##### Revision
Origin: 26ec2d4c08dc3b5a7ac9d2a8f7cf35b14ed4e2b6
Pull Request: 7a9126cba0171b76cddf7d64f7dfb23c81a719ee
Server: 7345fe34df791dd9f6dc9caddebfdd125bb6e8b2

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: f73badd8ce223662e575f68c02b0f607629a32ad
Pull Request: 6660f789292a8aefa19ff3c85ddbd38158705bd5
Server: f7c64caf98f4591a4c12ed26734916c427cef771

**Comment by ghost** [comment]
Ash lizards only get 1 puddle

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: 9514197e86ad105d345f1948f42145d76dd932ee
Pull Request: 29e4be76feb41db2ec7525ff332841b7f55413f4
Server: 7886f17d66d1531aedf01222d83c61c517ccfa02

File: code/modules/projectiles/guns/ballistic/automatic.dm [hunk 1]
```diff
@@ -26,10 +26,9 @@
 /obj/item/gun/ballistic/automatic/update_icon()//hippie edit -- bring back old gun icons
 	..()
 	if(!select)
-		add_overlay("[initial(icon_state)]semi")
+		add_overlay("[initial(icon_state)]_semi")
 	if(select == 1)
-		add_overlay("[initial(icon_state)]burst")
-	icon_state = "[initial(icon_state)][magazine ? "-[magazine.max_ammo]" : ""][chambered||!bolt_locked ? "" : "-e"][suppressed ? "-suppressed" : ""]"//hippie end -- bring back old gun icons
+		add_overlay("[initial(icon_state)]_burst")
```
Comment (yoyobatty) [inline code/modules/projectiles/guns/ballistic/automatic.dm]: This needs to be reverted

File: code/modules/projectiles/guns/ballistic/automatic.dm [hunk 2]
```diff
@@ -159,14 +158,13 @@
...
 	switch(select)
 		if(0)
-			add_overlay("[initial(icon_state)]semi")
+			add_overlay("[initial(icon_state)]_semi")
 		if(1)
-			add_overlay("[initial(icon_state)]burst")
+			add_overlay("[initial(icon_state)]_burst")
 		if(2)
 			add_overlay("[initial(icon_state)]gren")
 	icon_state = "[initial(icon_state)][magazine ? "" : "-e"]"
```
Comment (yoyobatty) [inline code/modules/projectiles/guns/ballistic/automatic.dm]: Fix this too

File: code/modules/projectiles/guns/ballistic/launchers.dm [hunk 3]
```diff
@@ -42,10 +42,6 @@
 	actions_types = list()
 	casing_ejector = FALSE
 
-/obj/item/gun/ballistic/automatic/gyropistol/update_icon()
-	..()
-	icon_state = "[initial(icon_state)][magazine ? "loaded" : ""]"
-
```
Comment (yoyobatty) [inline code/modules/projectiles/guns/ballistic/launchers.dm:48]: this too

File: code/modules/projectiles/guns/ballistic/toy.dm [hunk 4]
```diff
@@ -14,6 +13,10 @@
 	item_flags = NONE
 	casing_ejector = FALSE
 
+/obj/item/gun/ballistic/automatic/toy/update_icon()
+	. = ..()
+	add_overlay("[icon_state]_toy")
+
```
Comment (yoyobatty) [inline code/modules/projectiles/guns/ballistic/toy.dm]: this breaks it

File: code/modules/projectiles/guns/ballistic/toy.dm [hunk 5]
```diff
@@ -54,6 +56,10 @@
 	casing_ejector = FALSE
 	can_suppress = FALSE
 
+/obj/item/gun/ballistic/shotgun/toy/update_icon()
+	. = ..()
+	add_overlay("[icon_state]_toy")
+
```
Comment (yoyobatty) [inline code/modules/projectiles/guns/ballistic/toy.dm]: ditto

File: code/modules/projectiles/guns/ballistic/toy.dm [hunk 6]
```diff
@@ -90,10 +95,13 @@
 /obj/item/gun/ballistic/automatic/c20r/toy/unrestricted/riot
 	mag_type = /obj/item/ammo_box/magazine/toy/smgm45/riot
 
+/obj/item/gun/ballistic/automatic/c20r/toy/update_icon()
+	. = ..()
+	add_overlay("[icon_state]_toy")
+
```
Comment (yoyobatty) [inline code/modules/projectiles/guns/ballistic/toy.dm]: ditto

File: code/modules/projectiles/guns/ballistic/toy.dm [hunk 7]
```diff
@@ -106,3 +114,7 @@
 
 /obj/item/gun/ballistic/automatic/l6_saw/toy/unrestricted/riot
 	mag_type = /obj/item/ammo_box/magazine/toy/m762/riot
+
+/obj/item/gun/ballistic/automatic/l6_saw/toy/update_icon()
+	. = ..()
+	add_overlay("[icon_state]_toy")
```
Comment (yoyobatty) [inline code/modules/projectiles/guns/ballistic/toy.dm]: ditto

**Review by yoyobatty — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: d17f0a58560f900d6a15bb6947828fe6a9555d33
Pull Request: bef5ee926f65a0148582a56d82afd3416db667d1
Server: 721a72bd881333a7bd2426cd1fbb053965c7ef02

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: 2f1559a459a16a839534f1f69da49d009cc9959b
Pull Request: abc48fea04d76cb081f525de4e11a608c8375303
Server: 1ba41ae0317a287bf62f7fe1d87374e7ab85341d

**Comment by Carbonhell** [comment]
> Why are we upstream merging?

**Comment by cris5311** [comment]
This caused the removal of VR, right? If so, dont fucking merge it. Edit, i checked and yup it is.

**Comment by ghost** [comment]
@cris5311 we can readd it if we want. but didn't it cause alot of problems? like spamming ghosts?

**Comment by cris5311** [comment]
@steamp0rt i never got spammed by VR as ghost. It was acctually a thing which made dying quite bearable and when me and some other ghost found out it was removed after we got blown up in a clowncar in outerspace we were all very angry and sad.

**Comment by ghost** [comment]
@cris5311 also didn't some people lag the server via beam rifles from VR?
also having the window flash when alt+tabbed due to `Idiot McDumbFace has did blahblah at Syndicate VR` is very annoying.

**Comment by cris5311** [comment]
@steamp0rt i can see how that could be annoying. Maybe just remove the beamrifles from VR then? I never got the windowflash 

> when alt+tabbed due to `Idiot McDumbFace has did blahblah at Syndicate VR`

but i might have missed that due to not paying too much attention after im dead and deciding to not respawn as a ghost role or watch the game.

File: _maps/map_files/HippieStation/hippiestation.dmm [hunk 8]
```diff
@@ -273,7 +273,6 @@
 "aaV" = (
 /obj/machinery/conveyor{
 	dir = 5;
-	icon_state = "conveyor_map";
```
Comment (ghost) [inline _maps/map_files/HippieStation/hippiestation.dmm]: Um, did StrongDMM clean up default icon_states/invalid dirs?

**Comment by ghost** [comment]
@steamp0rt shouldn't designers and maintainers be the ones to make the final decisions on whether or not these changes to through?

**Comment by ghost** [comment]
I don't quite understand this whole PR

**Comment by Majick1** [comment]
I feel like there's just too much going on here for normal +- votes, leave this to the headmins/maintainers

**Comment by ghost** [comment]
@Majick1 I don't understand the whole concept of a mass merge. Why not simply mirror the PRs that you want instead?

**Comment by ghost** [comment]
If Hippie is going to go their own way they might as well go their own way

**Comment by Majick1** [comment]
This was started before HGTOW, and it looks like a bunch of map and menu fixes the mirror bot missed for whatever reason

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: fc0e0469fda3c8ddb56a6d566058f95cab67d249
Pull Request: 8b31621d89abb7c897668da91be36f1971687105
Server: 434460972ffaad89db6705f39cb26489cc4bc7b4

##### Comment
When I press this button, Steamport will die.

**Comment by optimumtact** [comment]
beautiful

**Comment by ghost** [comment]
@optimumtact begone thot

**Comment by ghost** [comment]
#### Test Merge Updated

##### Server Instance
HippieStation

##### Revision
Origin: fc0e0469fda3c8ddb56a6d566058f95cab67d249
Pull Request: 1271adbb040ca934ccafe01230f7304c280a626f
Server: 66be11ca1710b285d2bb1370fbaa5bdaae245453

##### Comment
hmm today I will replace sleepers with stasis beds

**Comment by AnthonyJNgo** [comment]
This is not to be merged until given headmin approved tag.

**Comment by AnthonyJNgo** [comment]
Last one.

**Comment by ghost** [comment]
wat?

**Comment by ghost** [comment]
#### Test Merge Removed

**Pull request merged**